libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Converters/DialogSetConverter.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
```

Comment:
license headers needed! 😊

Reply:
@cosmicshuai please add the headers, by the way, didn't our CI check that?

Reply:
license headers added